tinygrad/opsgpu.py

```diff
@@ -9,11 +9,14 @@ def buffer_new(ctx, shape):
   res_g.dtype = np.float32
   return res_g
 
+def buff(ctx, np_array):
+    res_g = cl.Buffer(ctx.cl_ctx, cl.mem_flags.WRITE_ONLY | cl.mem_flags.COPY_HOST_PTR, hostbuf=np_array)
```

Comment:
2 spaces here

Reply:
Heh, looks like I left my editor in 4 space per tab mode. Fixed.